.github/workflows/documentation.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Install apt dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install graphviz libclang1-11 libclang-cpp11
+        sudo apt-get install graphviz
```

Comment:
Are these now pre-installed?

Reply:
Yep, the clang-$VER packages now depend on them (e.g. https://code.launchpad.net/ubuntu/noble/amd64/clang-16/1:16.0.6-15, which is installed by default on the Ubuntu runners)